packages/tradershub/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModal.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { useActiveTradingAccount, useMT5AccountsList } from '@deriv/api';
+import { useBreakpoint } from '@deriv/quill-design';
+import { Dialog, Modal } from '../../../../components';
+import { Category, PlatformDetails } from '../../constants';
+import FooterComponent from './FooterComponent';
+import PasswordComponent from './PasswordComponent';
+
+const MT5PasswordModal = () => {
+    const [password, setPassword] = useState('');
+    const { data: activeTrading } = useActiveTradingAccount();
+    const { data: mt5Accounts } = useMT5AccountsList();
+    const { isMobile } = useBreakpoint();
+
+    const hasMT5Account = mt5Accounts?.find(account => account.login);
+    const isDemo = activeTrading?.is_virtual;
+
+    const ModalHeaderTitle = `${hasMT5Account ? 'Add' : 'Create'} a ${isDemo ? Category.DEMO : Category.REAL} ${
+        PlatformDetails.mt5.title
```

Comment:
One convention
CFDPlatforms or PlatformDetails

Reply:
the `PlatformDetails.mt5.title` has the word "Deriv" whereas `CFDPlatforms` is just 'mt5'